_test/test_algorithms/test_parallel_cut.m

```diff
@@ -21,13 +21,16 @@ function test_cut_cube_dummy(~)
             cut_ser = cut(data, proj, params{:});
             cut_par = parallel_call(@cut, {data, proj, params{:}});
 
+			% this should now work 1432 notwithstanding. Keeping the 2 comparisons so we can see
+			% which if either works
             assertEqualToTol(cut_ser.detpar, cut_par.detpar);
             % Re #1432 This is because something wrong with detector arrays
-            cut_ser.experiment_info.detector_arrays = cut_par.experiment_info.detector_arrays;
+            %cut_ser.experiment_info.detector_arrays = cut_par.experiment_info.detector_arrays;
             assertEqualToTol(cut_ser, cut_par, 'ignore_str', true,'-ignore_date')
         end
 
         function test_cut_cube_herbert(~)
+            skipTest('Job fails on Jenkins for unknown reasons see #1172')
```

Comment:
if it is fails on Jenkins only, it should be:
```
if is_jenkins()
  skipTest()
end
```

Reply:
true. inserted as shown by Jacob and fixed by him recently. now fixed here.